sweep/weight_estimator.go

```diff
@@ -119,5 +125,24 @@ func (w *weightEstimator) fee() btcutil.Amount {
 		fee = childFee
 	}
 
+	// Exit early if maxFeeRate is not set.
+	if w.maxFeeRate == 0 {
+		return fee
+	}
+
+	// Clamp the fee to the max fee rate.
+	maxFee := w.maxFeeRate.FeeForWeight(childWeight)
+	if fee > maxFee {
+		// Calculate the effective fee rate for logging.
+		childFeeRate := chainfee.SatPerKVByte(
+			fee * 1000 / btcutil.Amount(childWeight),
```

Comment:
I think this calculation is wrong because you're dividing by weight but the units are `sat/kvb`?

Reply:
so `fee / weight` is `sat/vb`, then `1 sat/vb = 1000/1000 sat/vb = 1000 sat/ 1000 vb = 1000 sat/kvb`?

Reply:
fee/weight is sat/wu

Reply:
switched to `kw`